frontends/parsers/parserDriver.cpp

```diff
@@ -156,6 +164,61 @@ P4ParserDriver::parse(FILE* in, const char* sourceFile,
     return parse(inputStream.get(), sourceFile, sourceLine);
 }
 
+template<typename T> const T*
+P4ParserDriver::parse(P4AnnotationLexer::Type type,
+                      Util::SourceInfo srcInfo,
+                      IR::Vector<IR::AnnotationToken>* body) {
+    if (Log::verbose())
+        std::cout << "Parsing P4-16 annotation " << srcInfo << std::endl;
```

Comment:
Generally prefer using `LOG` macros to checking Log::verbose() directly.  This will generate lots of cruft for people not interested in it.

Reply:
I've rewritten this to use `LOG3`, and rewritten the pre-existing checks to `Log::verbose()` to use `LOG1`.